backend/src/shared/sendEmail.ts

```diff
@@ -0,0 +1,39 @@
+import nodemailer from 'nodemailer';
+import logger from './Logger';
+import { config } from '../../config';
+
+class SendEmail {
+    private smtpTransport = nodemailer.createTransport({
+        service: 'gmail',
+        auth: {
+            user: config.mail.user,
+            pass: config.mail.pass
+        }
+    });
+
+    public sendPasswordRecoveryEmail(email: string, token: string) {
```

Comment:
We should try to not use string concat you can use  this notation insteaad `hello ${world}`

Reply:
ok